core/src/main/java/org/bitcoinj/script/ScriptBuilder.java

```diff
@@ -563,10 +563,10 @@ public static Script createCLTVPaymentChannelInput(byte[] from, byte[] to) {
         return builder.build();
     }
 
-    public static Script createCreditBurnOutput(ECKey creditBurnKey) {
+    public static Script createAssetLockOutput() {
         ScriptBuilder builder = new ScriptBuilder();
         builder.addChunk(new ScriptChunk(OP_RETURN, null));
-        builder.data(creditBurnKey.getPubKeyHash());
+        builder.data(new byte[0]);
         return builder.build();
     }
```

Comment:
The OP_RETURN will not have a reference to the key that will sign the state transitions that uses this asset lock.  instead this information is in the AssetLockPayload.